ui/src/track-layout/track-layout-react-utils.tsx

```diff
@@ -283,3 +291,61 @@ export function usePvDocumentHeader(
         [id, changeTime],
     );
 }
+
+export function refreshTrackNumberSelection(
+    onSelect: OnSelectFunction,
+    onUnselect: (items: OptionalUnselectableItemCollections) => void,
+): (id: LayoutTrackNumberId) => void {
+    return (id) => {
+        Promise.all([updateTrackNumberChangeTime(), updateReferenceLineChangeTime()]).then(
+            ([ts, _]) => {
+                getTrackNumberById(id, 'DRAFT', ts).then((tn) => {
```

Comment:
Onko näissä kaikissa tarkoituksella `DRAFT`? Ts. näitä ei koskaan haluttaisi käyttää ei-luonnostilasta?

Reply:
Mjoo, nykyiselläänhän sitä tietenkin käytetään vain draft-puolelta koska tarkoitus on refreshata valinnat kun tehdään muutos (ja mitään ei voi muuttaa officialissa). Mutta lienet oikeassa, samalla voisi ehkä jossain tilanteessa olla hyötyä official-siirtymässä ja vähintäänkin tätä koodia katsoessa ei ole ilmeistä miksi tämä on näin. Laitanpa selvyyden vuoksi tuon argumentiksi - siellä missä kutsu tehdään on ilmeistä että kiinnostaa vain DRAFT, toisin kuin täällä.